### ISSUE TITLE
A detail mistake in the paper

### ISSUE THREAD

**Issue by 1112321sfdsaf** [IsYuchenYuan/retinal-vessel-segmentation#1]
<img width="586" alt="image" src="https://user-images.githubusercontent.com/75245666/124060631-0de8ac80-da60-11eb-8a42-80d3c21071e0.png">
I guess it's "three".

**Comment by erwen-leuan** [comment]
This is an interesting work. I also have one concern on the paper. In Table VIII, cross experiments were made. When the model is trained on the STARE, and then tested on all DRIVE  dataset including the training set and test set or only one of both? Thank you very much for your answer.
